src/helpers.js

```diff
@@ -11,24 +11,39 @@
  * {one: {two: "bla"}}), `"value"` if the same key already exists swith a
  * different value, or `false`.
  */
-function dotPathToHash(entry, target = {}, options = {}) {
+function dotPathToHash(locale, entry, target = {}, options = {}) {
```

Comment:
I'm not keen with the `locale` argument to be the first one. I would tend to have it included in the options instead.

Reply:
Agreed.